app/models/community_translation.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
+#
+# Table name: community_translations
+#
+#  id              :integer          not null, primary key
+#  community_id    :integer          not null
+#  locale          :string(16)       not null
+#  translation_key :string(255)      not null
+#  translation     :string(255)
```

Comment:
Good point. Should these be text instead of varchars?

Or should we extend this structure when we add longer translations.
I.e. one option could be that we add separate table for 'long texts' and add just column to this table 
(some kind of long_text_ref_id)